components/fxa-client/src/oauth.rs

```diff
@@ -37,17 +38,27 @@ impl FirefoxAccount {
             }
         }
         let resp = match self.state.refresh_token {
-            Some(ref refresh_token) => {
-                if refresh_token.scopes.contains(scope) {
-                    self.client.oauth_token_with_refresh_token(
-                        &self.state.config,
-                        &refresh_token.token,
-                        &[scope],
-                    )?
-                } else {
-                    return Err(ErrorKind::NoCachedToken(scope.to_string()).into());
-                }
-            }
+//            Some(ref refresh_token) => {
+//                if refresh_token.scopes.contains(scope) {
+//                    self.client.oauth_token_with_refresh_token(
+//                        &self.state.config,
+//                        &refresh_token.token,
+//                        &[scope],
+//                    )?
+//                } else {
+//                    return Err(ErrorKind::NoCachedToken(scope.to_string()).into());
+//                }
+//            }
+            // TODO: FOR THE SAKE OF THE PR AND AS A ONE-TIME DEAL FOR VLAD,
+            // I REMOVED THE SCOPE CHECK.
```

Comment:
TODO: figure this thing out

Reply:
Is the problem that we need some scopes that weren't requested during the initial migration process? It looks like you're requesting "profile" and "oldsync" so it's not clear to me what other scopes you need.